src/ukify/ukify.py

```diff
@@ -521,21 +521,22 @@ def make_uki(opts):
     else:
         output = opts.output
 
-    objcopy_tool = find_tool('objcopy', opts=opts)
+    objcopy_tool = find_tool('llvm-objcopy', 'objcopy', opts=opts)
 
     cmd = [
         objcopy_tool,
         opts.stub,
         *itertools.chain.from_iterable(
-            ('--add-section',        f'{s.name}={s.content}',
-             '--change-section-vma', f'{s.name}=0x{s.offset:x}')
+            ('--add-section',       f'{s.name}={s.content}',
+             '--set-section-flags', f"{s.name}={','.join(s.flags)}")
             for s in uki.sections),
-        *itertools.chain.from_iterable(
-            ('--set-section-flags',  f"{s.name}={','.join(s.flags)}")
-            for s in uki.sections
-            if s.flags is not None),
         output,
     ]
+
+    if pathlib.Path(objcopy_tool).name == 'objcopy':
```

Comment:
I'd recommend making this `!= llvm-objcopy` instead - reason being, usually to do cross-arch objcopy the binutils package might provide foreign target builds. For example on Debian you can install `binutils-aarch64-linux-gnu` which provides `aarch64-linux-gnu-objcopy` which would need `--change-section-vma`